### DISCUSSION TITLE
Please update Corepack to 0.27 or 0.28

### DISCUSSION THREAD

**Discussion by onigoetz** [renovatebot/renovate#28582] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

Hi,
recently Renovate updated to Corepack 0.26.0 which contains a bug that breaks it when using a corporate proxy. ( https://github.com/nodejs/corepack/issues/417 ). This prevents us from running the latest Renovate versions

This bug was recently fixed ( https://github.com/nodejs/corepack/pull/451 ) and released ( https://github.com/nodejs/corepack/releases/tag/v0.27.0 )

Can you please include the newest version so that we can update to the latest Renovate version on our side ?

Thanks in advance

### Logs (if relevant)

<details><summary>Logs</summary>

```

N/A

```

</details>

**Comment by rarkins** [comment, marked as answer]
https://github.com/renovatebot/base-image/pull/391
